spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-lattice/pom.xml

```diff
@@ -27,6 +27,7 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-lattice-core</artifactId>
 			<version>${spring-cloud-lattice.version}</version>
+			<scope>runtime</scope>
```

Comment:
Scratch that, chances are we can't go "runtime" because `@Configuration` requires compile-time visibility